static/docs/commands-reference/add.md

```diff
@@ -2,82 +2,193 @@
 
 Take a data file or a directory under DVC control.
 
+## Synopsis
+
 ```usage
-    usage: dvc add [-h] [-q] [-v] [-R] [--no-commit] targets [targets ...]
+    usage: dvc add [-h] [-q | -v] [-R] [--no-commit] targets [targets ...]
 
     positional arguments:
-      targets               Input files/directories
-
-    optional arguments:
-      -h, --help            show this help message and exit
-      -q, --quiet           Be quiet.
-      -v, --verbose         Be verbose.
-      -R, --recursive       Recursively add each file under the directory.
-      --no-commit           Don't save changes to cache.
+      targets          Input files/directories.
+
 ```
 
-Under the hood a few steps are happening:
+## Description
+
+The `dvc add` command is analogous to the `git add` command.  By default an 
+added file is committed to the DVC cache.  Using the `--no-commit` option, the
+file will not be added to the cache and instead the `dvc commit` command is
+used when (or if) the file is to be committed to the DVC cache.
+
+Under the hood a few actions are taken for each file in the target(s):
 
 1. Move the file content to the DVC cache (default location is `.dvc/cache`).
 2. Calculate the file checksum.
 3. Replace the file by a link to the file in the cache (see details below).
 4. Create a corresponding DVC file (metafile `.dvc`) and store the checksum
-to identify the right file in cache.
-
-Only _metafile_ (basically, pointer to the data in cache) is stored in Git,
-DVC manages data file contents.
-
-See [DVC File Format](/doc/user-guide/dvc-file-format) for the detailed
-description of the _metafile_ format.
-
-DVC stores the file's last modification timestamp, inode, and the checksum into
-a global state file `.dvc/state` to reduce time recomputing checksums later.
-
-Note, by default dvc tries a range of link types (reflink, hardlink, symlink,
-copy) to try to avoid copying any file contents and make dvc file operations
-very quick even for large files. Reflink is the best link type we could have,
-but even though it is becoming more and more common in modern filesystems, many
-filesystems still don't support it and thus dvc has to resort to a much more
-common hardlinks. See `dvc config` for more information.
+  to identify the cache entry.
+5. Add the _target_ filename to `.gitignore` to prevent it from being 
```

Comment:
DVC can run w/o SCM (git) or on top of other SCM (Mercurial support PR is in progress). Should we at least put a something like `(if git is used)`?

Reply:
If there's no Git does it still set up `.gitignore`?

Reply:
I'm not sure. There us `dvc init --no-scm` to initialize it w/o SCM. If keeps creating .gitignore it's a bug I would say and we need to create a ticket for that.

Reply:
I am replicating using `--no-scm` and after `dvc add data` it printed

```
To track the changes with git run:

	git add data.dvc
```

Reply:
No `.gitignore` is created though

Reply:
Yep, let's create an issue here - https://github.com/iterative/dvc/issues!

Reply:
Put together an issue - https://github.com/iterative/dvc/issues/1685